modules/oe_whitelabel_multilingual/oe_whitelabel_multilingual.module

```diff
@@ -39,20 +39,24 @@ function oe_whitelabel_multilingual_preprocess_links__oe_multilingual_content_la
   $entity = $multilingual_helper->getEntityFromCurrentRoute();
   /** @var \Drupal\Core\Entity\EntityInterface $translation */
   $translation = $multilingual_helper->getCurrentLanguageEntityTranslation($entity);
-  $variables['languages'][] = [
-    'path' => $translation->toUrl()->setAbsolute(TRUE)->toString(),
-    'hreflang' => $translation->language()->getId(),
-    'label' => $languages[$translation->language()->getId()]->getName(),
-    'current' => TRUE,
-    'icon_position' => 'before',
-    'remove_icon_spacers' => FALSE,
-    'icon' => [
-      'name' => 'check-lg',
-      'path' => $variables['bcl_icon_path'],
-      'size' => 'xs',
-      '#attributes' => ['class' => ['me-2']],
-    ],
-  ];
+  // If we don't have a language id defined yet, the current translation wasn't
+  // saved, so we don't add it to the list.
+  if ($translation->language()->getId() !== 'und') {
```

Comment:
I notice you are using `'und'` string literal instead of a constant.
In Drupal 7 I would have told you to use `LANGUAGE_NONE`.
Now this has changed, see [LANGUAGE_NONE changed to LANGUAGE_NOT_SPECIFIED, LANGUAGE_NOT_APPLICABLE and LANGUAGE_MULTIPLE added
](https://www.drupal.org/node/1473200)
Not sure if we also need to support other possible values.
Perhaps safer to just check `isset($languages[$current_content_language_id])`?